force_nevergrad/mco/ng_mco_model.py

```diff
@@ -27,6 +29,3 @@ def default_traits_view(self):
             Item("budget", label="Allowed number of objective calls"),
             Item("verbose_run"),
         )
-
-    def _algorithms_default(self):
```

Comment:
Was this looked at? The behaviour we want is for the default algorithm to be the same each time an instance of `NevergradMCOModel` is created. Otherwise if this isn't specified by the user they may get unexpected results.

It's possible that the `ng.optimizers.registry` dictionary is ordered, but it could also be changes during a version update.

Reply:
Ah, I can see that the `NevergradScalarOptimizer` and `NevergradMultiOptimizer` have explicit default values for the `algorithms` attribute, but this just needs to also be carried through to the `NevergradMCOModel` class